project.clj

```diff
@@ -66,7 +66,10 @@
                  [ring/ring-jetty-adapter "1.4.0"]                    ; Ring adapter using Jetty webserver (used to run a Ring server for unit tests)
                  [ring/ring-json "0.4.0"]                             ; Ring middleware for reading/writing JSON automatically
                  [stencil "0.5.0"]                                    ; Mustache templates for Clojure
-                 [swiss-arrows "1.0.0"]]                              ; 'Magic wand' macro -<>, etc.
+                 [swiss-arrows "1.0.0"]                               ; 'Magic wand' macro -<>, etc.
+                 [io.crate/crate-jdbc "1.11.0"]
+                 [io.crate/crate-client "0.54.7"]]
```

Comment:
Mind moving these up a bit? We keep all of Metabase's dependencies sorted by name. A comment explaining its purpose like the other dependencies would be helpful as well.